src/ucp/core/ucp_mm.c

```diff
@@ -1561,7 +1561,7 @@ ucp_memh_import_slow(ucp_context_h context, ucs_rcache_t *existing_rcache,
             ucs_snprintf_safe(rcache_name, sizeof(rcache_name),
                               "ucp_import_rcache[0x%" PRIx64 "]",
                               unpacked->remote_uuid);
-            status = ucp_mem_rcache_create(context, rcache_name, &rcache);
+            status = ucp_mem_rcache_create(context, rcache_name, &rcache, NULL);
```

Comment:
IMOP ucp_mem_rcache_create() function better accept rcache_params instead of reache_config. and ucp_mem_rcache_init() will translate config to params.
this way no need to handle the special case of "NULL" in ucs_rcache_set_params(), and instead set them here (just for the imported keys rcache)